content/docs/user-guide/components/reference/audio/switch.md

```diff
@@ -11,13 +11,10 @@ The **Audio Switch** component provides basic [Audio Translation Layer (ATL)](/d
 
 ## Audio Switch Properties
 
-The Audio Switch component has the following properties:
-
-**Default Switch**
-Enter the name of the audio switch to use by default. You can associate any audio switch with the entity.
-
-**Default State**
-Enter the name of the audio switch state to use by default. Use the [Audio Controls Editor](/docs/user-guide/interactivity/audio/audio-controls-editor) to assign the state to the switch. When this component is activated, the default switch is set to the default state.
+| Name | Description | Default |
+|------|-------------|---------|
+| Default Switch | Enter the name of the audio switch to use by default. You can associate any audio switch with the entity. | `<Empty>` |
+| Default State | Enter the name of the audio switch state to use by default. Use the [Audio Controls Editor](/docs/user-guide/interactivity/audio/audio-controls-editor) to assign the state to the switch. When this component is activated, the default switch is set to the default state. | `<Empty>` |
```

Comment:
```suggestion
| **Default State** | Enter the name of the audio switch state to use by default. Use the [Audio Controls Editor](/docs/user-guide/interactivity/audio/audio-controls-editor) to assign the state to the switch. When this component is activated, the **Default Switch** is set to the **Default State**. | `<Empty>` |
```
Bold elements of the UI, including property names.  We can remind users that this property is affected by another property's value by using UI element's names with bolding again in the description.